controllers/statuscheck/conditions_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package statuscheck
+
+import (
+	"testing"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+)
+
+func Test_isThresholdExceed(t *testing.T) {
+	type args struct {
+		records   []v1alpha1.StatusCheckRecord
+		want      v1alpha1.StatusCheckOutcome
```

Comment:
maybe change a name other than `want`, it is a little ambiguous with following `want` in `tests`(out of `args`)

Reply:
Actually it was auto-generated by GoLand 😢, do you have any suggestions to instead `want`?

Reply:
Fixed! (I found that I misunderstand your point before 😢)